packages/@ember/-internals/glimmer/tests/integration/components/append-test.js

```diff
@@ -37,7 +38,9 @@ class AbstractAppendTest extends RenderingTestCase {
     this.ids.push(component.elementId);
   }
 
-  ['@test lifecycle hooks during component append'](assert) {
+  [`${testUnless(
+    DEPRECATIONS.DEPRECATE_COMPONENT_TEMPLATE_RESOLVING.isRemoved
+  )} lifecycle hooks during component append`](assert) {
```

Comment:
This seems like test coverage that we would still want to keep after removing the deprecation?

Reply:
I can't figure this out -- test is too tied to specific VM behaves, it seems

Reply:
I did figure something out by not using any of the same test apis, and requiring slightly different assertions due to using the whole rendering system instead of manually creating factories, but I think the intent of the test is maintained in the new way